client/src/templates/Introduction/intro.css

```diff
@@ -138,7 +138,7 @@ button.map-title {
   padding-inline-start: 15px;
 }
 
-.block .map-cert-title {
+.cert-block .map-cert-title {
```

Comment:
`block` is a classname that affect tailwind, needed to change it to something that doesn't hook to it